### ISSUE TITLE
Update Ukrainian translations

### ISSUE THREAD

**Issue by alina-yur** [squidfunk/mkdocs-material#4943]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### Language availability

- [X] I've checked the [list of available languages](https://bit.ly/33vFDD0)

### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "uk",
  "action.edit": "Редагувати сторінку",
  "action.skip": "Перейти до змісту",
  "action.view": "Переглянути вихідний код сторінки ⬅️",
  "announce.dismiss": "Більше не показувати ⬅️",
  "blog.archive": "Архівувати ⬅️",
  "blog.categories": "Категорії ⬅️",
  "blog.categories.in": "в ⬅️",
  "blog.continue": "Читати далі ⬅️",
  "blog.draft": "Чернетка ⬅️",
  "blog.index": "Повернутись на головну ⬅️",
  "blog.meta": "Метадані ⬅️",
  "blog.references": "Пов'язані посилання ⬅️",
  "clipboard.copy": "Скопіювати в буфер",
  "clipboard.copied": "Скопійовано в буфер",
  "consent.accept": "Прийняти ⬅️",
  "consent.manage": "Керувати налаштуваннями ⬅️",
  "consent.reject": "Відхилити ⬅️",
  "footer": "Футер ⬅️",
  "footer.next": "Вперед",
  "footer.previous": "Назад",
  "header": "Хедер ⬅️",
  "meta.comments": "Коментарі",
  "meta.source": "Вихідний код",
  "nav": "Навігація ⬅️",
  "readtime.one": "Час на прочитання: 1 хвилина ⬅️",
  "readtime.other": "Час на прочитання: # хвилин ⬅️",
  "rss.created": "RSS стрічка ⬅️",
  "rss.updated": "RSS стрічка оновленого контенту ⬅️",
  "search": "Шукати ⬅️",
  "search.placeholder": "Пошук",
  "search.share": "Поділитись ⬅️",
  "search.reset": "Очистити ⬅️",
  "search.result.initializer": "Пошук розпочато ⬅️",
  "search.result.placeholder": "Розпочніть писати для пошуку",
  "search.result.none": "Збігів не знайдено",
  "search.result.one": "Знайдено 1 збіг",
  "search.result.other": "Знайдено # збігів",
  "search.result.more.one": "Ще 1 збіг на цій сторінці ⬅️",
  "search.result.more.other": "Ще # збігів на цій сторінці ⬅️",
  "search.result.term.missing": "Не знайдено запиту ⬅️",
  "select.language": "Обрати мову ⬅️",
  "select.version": "Обрати версію ⬅️",
  "source": "Перейти до вихідного коду",
  "source.file.contributors": "Контриб'ютори ⬅️",
  "source.file.date.created": "Створено",
  "source.file.date.updated": "Востаннє оновлено",
  "tabs": "Вкладки ⬅️",
  "toc": "Зміст",
  "top": "Повернутись нагору ⬅️"
}[key] }}{% endmacro %}
```

**Comment by alina-yur** [comment]
Hi. I did my best to come with accurate translations, but for some phrases I was missing context and I couldn't seem to find it in the repo. So happy to work on improvements if necessary once this is rolled out or issues are reported.

**Comment by squidfunk** [comment]
Thanks! We're working on providing more context as part of this issue template! We'll use your translations as they are right now, but can also improve them at any time 😊

**Comment by alina-yur** [comment]
@squidfunk thank you for making it so easy to contribute and caring about all the languages:) ❤️

**Comment by squidfunk** [comment]
57, and counting! 🚀 Added in 664af42367c696abf0d7f58e3903fb44579cde89.

**Comment by squidfunk** [comment]
Released as part of [9.0.8](https://github.com/squidfunk/mkdocs-material/releases/tag/9.0.8).
